salesagility/SuiteCRM
# Feature/massupdate extendable custom

Adds a helper function to load a class from custom if it is extended, otherwise from core.
Modifies calls to MassUpdate to use this helper function.

## Motivation and Context

Makes MassUpdate.php extendable through custom.

## How To Test This


1. Create a custom MassUpdate file in /custom with a class that either extends or replaces the core class.
2. See it loads correctly.

## Types of changes


### Final checklist
